### PR TITLE
Fix an issue where selectors are not showing current selection

### PR DESCRIPTION
**Pull request by JoaquinTrinanes**
Fixes select value not being shown when the value changed.
I tried adding some type safety to avoid this kind of bug in the future (`useForm<Step1Props>`), but I'm not exactly sure of the types of the other form elements and would need to check it out.

image

### CODE REVIEW

File: client/src/containers/interventions/new/step1/component.tsx [hunk 1]
```diff
@@ -168,7 +168,7 @@ const Step1: FC = () => {
             options={optionsBusinesses}
             placeholder="all businesses"
             theme="inline-primary"
-            onChange={({ value }) => handleDropdown('businessUnitsIds', value)}
+            onChange={(value) => handleDropdown('businessUnitsIds', value)}
```
Comment (mluena) [inline client/src/containers/interventions/new/step1/component.tsx:171]: when you open the form for a new intervention the default option for this select should be "all business". Also, if you change the params for handleDropdown, you need to set the values differently in the function as we are saving just the business id (just the value of the option selected)

File: client/src/containers/interventions/new/step1/component.tsx [hunk 2]
```diff
@@ -107,6 +109,8 @@ const Step1: FC = () => {
     dispatch(resetInterventionData());
   }, [dispatch]);
 
+  console.log(watch());
```
Comment (davidsingal) [inline client/src/containers/interventions/new/step1/component.tsx:112]: do not forget to remove console.log

**Review by davidsingal — CHANGES_REQUESTED** [review]

**Comment by davidsingal** [comment]
It's still not working, It's still failing on the selects of step 2.

File: client/src/containers/interventions/new/step1/component.tsx (hunk 1 above)
Comment (JoaquinTrinanes) [reply client/src/containers/interventions/new/step1/component.tsx:189]: You're right, that's already taken care of. Next time I'll check that all data is stored correctly, promise!

File: client/src/containers/interventions/new/step1/component.tsx (hunk 2 above)
Comment (JoaquinTrinanes) [reply client/src/containers/interventions/new/step1/component.tsx]: FYI ;) https://eslint.org/docs/rules/no-console

**Comment by JoaquinTrinanes** [comment]
> It's still not working, It's still failing on the selects of step 2.

@davidsingal Should be fixed now!

**Review by davidsingal — APPROVED** [review]

**Pull request merged**